lldb/test/API/functionalities/progress_reporting/TestProgressReporting.py

```diff
@@ -25,8 +25,8 @@ def test_dwarf_symbol_loading_progress_report(self):
         event = lldbutil.fetch_next_event(self, self.listener, self.broadcaster)
         ret_args = lldb.SBDebugger.GetProgressFromEvent(event)
         self.assertGreater(len(ret_args), 0)
-        message = ret_args[0]
-        self.assertGreater(len(message), 0)
+        title = ret_args[0]
```

Comment:
what's the point of this change ?

Reply:
In older versions of `Progress` the message that got displayed to the user was called `message` but it was changed to `title` when progress reports got updated to have `details` as a field, so I changed the name in the test to match.

Reply:
I know, my point was that this change is unrelated to this PR and also NFC, which basically means we're re-writing the git history for no good reason. We try to avoid that in general.

Reply:
Ah okay, I'll remove this change then.